test/test_fenwick_tree.cpp

```diff
@@ -68,7 +68,7 @@ void test_fenwick_sum_onebased() {
     using namespace Fenwick_Sum_Onebased;
```

Comment:
The naming should be consistent. We have FenwickTree 1-based as the default, so the other one should be called 0-based.